9/part2.js

```diff
@@ -0,0 +1,24 @@
+const { input } = require('./input');
+
+const sum = nums => nums.reduce((t, i) => t + i, 0);
+
+function contiguousSum(nums, target) {
+  for(let i = 0; i < nums.length; i++) {
+    for(let j = i+1; j < nums.length; j++) {
+      let range = nums.slice(i, j);
+      if(sum(range) === target) {
+        return range;
+      }
+    }
+  }
+}
```

Comment:
You can get `O(n)` here with a sliding window. E.g. https://github.com/neverendingqs/advent-of-code/pull/9/files#diff-edc2045fa6a970fcfffdc2873b4170fbea2db0b7505b94a4d715acd97f536c18R71-R83